org.omg.sysml/src/org/omg/sysml/expressions/AtFunction.java

```diff
@@ -0,0 +1,10 @@
+package org.omg.sysml.expressions;
```

Comment:
This file is missing the copyright header. I am not sure how important it is here, but the other classes of this package include the header, so I wanted to mention it.

Reply:
I added the missing header.